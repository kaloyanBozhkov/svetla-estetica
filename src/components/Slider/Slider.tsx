import type { ReactElement } from 'react'
import styles from './slider.module.scss'

// import slides
import Slide1 from './Slides/Slide1'

const slides = [Slide1],
    Slider = (): ReactElement => (
        <div className={styles.slider}>
            {slides.map((Slide) => (
                <Slide key={Slide.name} />
            ))}
        </div>
    )

export default Slider
