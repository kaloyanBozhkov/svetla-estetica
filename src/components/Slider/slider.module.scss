.slider {
    height: 45rem;
    width: 100%;
    white-space: nowrap;
}
