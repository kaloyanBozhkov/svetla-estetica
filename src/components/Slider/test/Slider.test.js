import { shallow } from 'enzyme'
import React from 'react'
import Slider from '../Slider'

describe('testing Slider', () => {
    it('should render component', () => {
        expect(shallow(<Slider />)).toMatchSnapshot()
    })
})
