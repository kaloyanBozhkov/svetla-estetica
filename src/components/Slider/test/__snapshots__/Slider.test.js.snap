// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Slider should render component 1`] = `
<div
  className="slider"
>
  <Slide1
    key="Slide1"
  />
</div>
`;
